core/src/main/scala/com/metamx/tranquility/beam/HttpBeam.scala

```diff
@@ -154,13 +157,13 @@ class HttpBeam[A: Timestamper](
                 "eventCount" -> events.size
               )
             )
-            Future.value(0)
+            Future.value(BitSet.empty)
```

Comment:
sorta. https://github.com/druid-io/tranquility/blob/master/docs/overview.md#guarantees says,

> If there is an issue that prevents Tranquility from receiving an acknowledgement from the indexing service, it will retry the batch, which can lead to duplicated events.

will address this and one of your other comments by updating the Beam and Tranquilizer interface docs.